data_steward/deid/aou.py

```diff
@@ -161,7 +161,7 @@ def create_person_id_src_hpo_map(client, input_dataset):
         sql_statement.append(
             sql.format(table=table, input_dataset=input_dataset))
 
-    final_query = ' UNION ALL '.join(sql_statement)
+    final_query = ' UNION '.join(sql_statement)
```

Comment:
This might be better as ` UNION DISTINCT `.  Thoughts?

Reply:
I thought `UNION` and `UNION DISTINCT` were the same. They both remove the duplicate from the output.
Do they behave differently?

Reply:
This used to be ` UNION ALL `.  When I try the following in the GCP UI, it errors out.
```sql
select 'foo' as name
union
select 'bar' as name
union
select 'foo' as name;
```

I have to specify either ` UNION ALL ` or ` UNION DISTINCT `.

Reply:
Thanks for pointing it out! I did not know `UNION` does not work with BigQuery 😮 Let me update it.